superset/migrations/versions/2022-07-11_11-26_ffa79af61a56_rename_report_schedule_extra_to_extra_.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""rename_report_schedule_extra_to_extra_json
+
+So we can reuse the ExtraJSONMixin
+
+Revision ID: ffa79af61a56
+Revises: a39867932713
+Create Date: 2022-07-11 11:26:00.010714
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "ffa79af61a56"
+down_revision = "a39867932713"
+
+from alembic import op
+from sqlalchemy.types import Text
+
+
+def upgrade():
+    op.alter_column(
+        "report_schedule",
+        "extra",
+        new_column_name="extra_json",
```

Comment:
Why the rename? Is this consistent with other tables. I've always felt that the `extra` column was always implicitly interpreted as being JSON encoded.

Reply:
This is for compatibility with the [`ExtraJSONMixin`](https://github.com/apache/superset/blob/2c010974e8c3607ff19d864c9edd910b1f7920e8/superset/models/helpers.py#L562), which seems to be one way of ensuring consistency. We should probably migrate all models to use this mixin.